packages/editor/src/browser/editor.ts

```diff
@@ -36,6 +36,8 @@ export interface TextEditor extends Disposable, TextEditorSelection, Navigatable
     selection: Range;
     readonly onSelectionChanged: Event<Range>;
 
+    readonly onEditorOptionsChanged: Event<any>;
```

Comment:
either don't expose it here and do completely in the monaco extension or please write proper typings,  `any` does not make any good in public apis

Reply:
also to make this part of api compete, there should be `updateOptions` and `getOptions`, an event should be renamed to `onOptionsChanged`, since you are already in the editor context, there is no reason to repeat yourself with `onEditorOptionsChanged`